setup.cfg

```diff
@@ -39,12 +39,6 @@ python_requires = >=3.9
 include_package_data = True
 
 install_requires =
```

Comment:
If this is empty, is this field needed?

Reply:
Boilderplate code - left in other packages too.